tests/func/test_get.py

```diff
@@ -87,3 +89,38 @@ def test_get_to_dir(dname, erepo):
 
     assert os.path.isdir(dname)
     assert filecmp.cmp(erepo.FOO, dst, shallow=False)
+
+
+def test_get_from_non_dvc_master(empty_dir, git_erepo, caplog):
```

Comment:
How about using `erepo` just removing `.dvc` dir at the start of the test?

Reply:
I can do that, but test won't be much more readable due to fact that the default remote set inside `erepo` creation is its own cache and I need to provide a new one during this particular test.

Reply:
After giving it some thought, that is not possible. Such a case would defeat the test purpose. External repo is cloning the target, so it doesn't matter that I remove `.dvc` from `erepo`, it will still be `dvc` repo after clone.

Reply:
If you remove `.dvc` and commit that then that won't be a dvc repo :) Now you make a lot of manipulations.